gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dform_ttt2.lua

```diff
@@ -640,7 +640,7 @@ derma.DefineControl("DFormTTT2", "", PANEL, "DCollapsibleCategoryTTT2")
 -- @return Panel The created collapsable form
 -- @realm client
 function vgui.CreateTTT2Form(parent, name)
-	local form = vgui.Create("DFormTTT2", parent)
+	local form = vgui.Create("DFormTTT2", parent, name)
 
 	form:SetName(name)
```

Comment:
This would be redundant then, right?

Definitely weird that this seemingly does not set the name of the panel ...

Reply:
i think SetName changes the display string somewhere down the line, but I need to pass it when vguiCreate is run for then to not be named dpanel, i tested without the SetName and that is what I observed